### ISSUE TITLE
imx8mp usb dwc3 clock quirk needed in DT

### ISSUE THREAD

**Issue by hcl-dr** [barebox/barebox#19]
Quirk is missing from the kernel imported imx8mp.dtsi (since 2022.12.0)

Symptom: panics with USB memory stick

```
[<bfd01d08>] (panic+0x5c/0x78) from [<bfd30dac>] (xhci_wait_for_event+0xb8/0x11c)
[<bfd30dac>] (xhci_wait_for_event+0xb8/0x11c) from [<bfd2f3f4>] (xhci_submit_control_msg+0x554/0xab0)
[<bfd2f3f4>] (xhci_submit_control_msg+0x554/0xab0) from [<bfd2625c>] (usb_control_msg+0xb0/0xec)


&usb_dwc3_0 {
	snps,dis-u2-freeclk-exists-quirk;
};

&usb_dwc3_1 {
	snps,dis-u2-freeclk-exists-quirk;
};
```

**Comment by a3f** [comment]
This should be resolved by 5d01c3317565 ("usb: dwc3: sync with Linux-6.3-rc2"). This is not yet in a release, but it's in [next](https://github.com/saschahauer/barebox/tree/next).

**Comment by a3f** [comment]
A workaround for v2023.04.0: https://lore.barebox.org/barebox/20230330113844.3343865-1-a.fatoum@pengutronix.de

**Comment by a3f** [comment]
This has been fixed in v2023.04.0.
